.github/workflows/master-build.yml

```
@@ -50,21 +53,30 @@ jobs:
           - 9001:9001
-        options: -e="MINIO_ROOT_USER=root" -e="MINIO_ROOT_PASSWORD=password" -e="MINIO_DEFAULT_BUCKETS=default"
+        options: >-
+          -e="MINIO_ROOT_USER=root"
+          -e="MINIO_ROOT_PASSWORD=password"
+          -e="MINIO_DEFAULT_BUCKETS=default"
 
     steps:
-      - name: Test Database
+      - name: Test Elasticsearch health
         env:
           ELASTIC_SEARCH_URL: http://localhost:${{ job.services.elasticsearch.ports[9200] }}
```

actionlint (1.7.12)

[error] property access of object must be type of string but got "number"

(expression)